playbooks/juju/post.yaml

```diff
@@ -38,13 +38,13 @@
         owner: no
         group: no
     - name: 'force remove Juju machines'
-      command: juju remove-machine --force `seq 0 100`
+      shell: juju remove-machine --force `seq 0 100`
```

Comment:
here you are still using backticks but on the other you are using `$()`. Can we align?